rosapi/interface.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+
+class Interface():
+  def __init__(self, api, interface):
+    self.interface_name = interface
+    self.api = api
+
+  def checkinterface(self):
```

Comment:
I get the checkinterface naming, but I think here we should also keep things consistent and use the exact naming conventions of upstream. 

so, since we're already in a Class named 'Interface', there's abosolutely no need to redundantly use the word interface in the methods (this is a common mistake).  

interface.print matches perfectly to the cli: 
/interface print